.circleci/config.yml

```diff
@@ -1,76 +1,97 @@
 version: 2.1
 
-var_1: &working_dir ~/src
-var_2: &dapp_working_dir ~/src/raiden-dapp
+parameters:
+  base_executor_image_tag:
+    type: string
+    default: lts-browsers
+
+  e2e_environment_executor_image_tag:
+    type: string
+    default: latest
+
+  working_directory:
+    type: string
+    default: ~/src
+
+  test_report_directory:
+    type: string
+    default: reports/junit
 
 anchor_1: &attach_options
-  at: *working_dir
+  at: << pipeline.parameters.working_directory >>
 
 anchor_2: &executor_parameter
   parameters:
     executor:
       type: executor
       default: base-executor
 
-anchor_3: &run_on_release_tag_only
+anchor_3: &executor_shared_options
+  working_directory: << pipeline.parameters.working_directory >>
+  environment:
+    JEST_JUNIT_OUTPUT_DIR: << pipeline.parameters.test_report_directory >>
+
+anchor_4: &filter_release_tag
   filters:
     tags:
       only: /^v\d+\.\d+\.\d+$/
-    branches:
-      ignore: /.*/
 
 executors:
   base-executor:
-    working_directory: *working_dir
+    <<: *executor_shared_options
     docker:
-      - image: circleci/node:lts-browsers
+      - image: cimg/node:<< pipeline.parameters.base_executor_image_tag >>
```

Comment:
Nitpick: I don't think putting only the tag on a parameter is very useful, it's more probable to change the whole image than to switch tags. Wouldn't be worth it to put the whole image in the parameter instead?

Reply:
Sure we can do that. :+1:

Reply:
:heavy_check_mark: